### PR TITLE
Emergency shuttle information display improvements

### PR DESCRIPTION
**Pull request by timothymtorres**
## About The Pull Request
Emergency shuttles now:

- Display occupancy limits and prerequisites  
- Has their own folder that is split into several subfiles
- Have documentation for a few variables

## Why It's Good For The Game
4cUSVwSNl8

## Changelog
:cl:
qol: Emergency shuttle now gracefully display occupancy limits and prerequisites in the communication console.
/:cl:

### CODE REVIEW

File: code/datums/shuttles/whiteship.dm [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+/datum/map_template/shuttle/whiteship/delta
+	suffix = "delta"
+	name = "NT Frigate"
+	description = "A standard whiteship with big spiders onboard. PACMAN generator is not wired and next to main grid cabling, so it requires some work."
+
+/datum/map_template/shuttle/whiteship/pod
+	suffix = "whiteship_pod"
+	name = "Salvage Pod"
+	description = "There is no map for this vessel and it was supposed to be used with the Meta-class. Do not try to spawn it!"
```
Comment (Ghommie) [inline code/datums/shuttles/whiteship.dm:52]: We no longer have the whiteship pod, right? So why are we keeping this template arond?

**Review by Ghommie — APPROVED** [review]
All good but one no longer used template that caught my eye.

File: code/datums/shuttles/whiteship.dm (hunk 1 above)
Comment (timothymtorres) [reply code/datums/shuttles/whiteship.dm]: Removed with https://github.com/tgstation/tgstation/pull/77457/commits/5b10fa4b3079926dbf925d6c66135c6ed7bc6b4e

Also went and removed the two useless pod computers from two whiteships and replaced it with tables.  This should give more space to the cockpits.

**Comment by Ghommie** [comment]
Waiting on you to solve the conflicts, then it's gucci.

**Comment by timothymtorres** [comment]
Ready to go!

**Pull request merged**